test/e2e/framework/pv/pv.go

```diff
@@ -676,7 +678,7 @@ func WaitForPVClaimBoundPhase(client clientset.Interface, pvclaims []*v1.Persist
 	persistentvolumes := make([]*v1.PersistentVolume, len(pvclaims))
 
 	for index, claim := range pvclaims {
-		err := WaitForPersistentVolumeClaimPhase(v1.ClaimBound, client, claim.Namespace, claim.Name, Poll, timeout)
+		err := framework.WaitForPersistentVolumeClaimPhase(v1.ClaimBound, client, claim.Namespace, claim.Name, framework.Poll, timeout)
```

Comment:
In the spirit of stripping away any PV-related functionality from the framework pkg (`/test/e2e/framework`).
I think it would be best to replace this call to `framework.WaitForPersistentVolumeClaimPhase` with the code being called by it.

https://github.com/kubernetes/kubernetes/blob/8b4fd4104df3374891e3a538eac4c4bdbd06719e/test/e2e/framework/util.go#L729-L731

`framework.WaitForPersistentVolumeClaimPhase` is just a wrapper for [WaitForPersistentVolumeClaimsPhase](https://github.com/kubernetes/kubernetes/blob/8b4fd4104df3374891e3a538eac4c4bdbd06719e/test/e2e/framework/util.go#L735).
We should move this method into `/test/e2e/framework/pv`.